packages/app/cypress/e2e/cypress-in-cypress-run-mode.cy.ts

```diff
@@ -50,7 +56,13 @@ describe('Cypress In Cypress - run mode', { viewportWidth: 1200 }, () => {
     cy.contains('500x500').should('be.visible')
     cy.contains('Chrome 1').should('be.visible')
 
-    cy.percySnapshot()
+    cy.percySnapshot({
+      elementOverrides: {
```

Comment:
It would be nice to move this override to the `installCustomPercyCommand` so this doesn't crop up again in future diffs.

Reply:
good idea, let's see if it work